### DISCUSSION TITLE
After finetuning with Autotrain, it doesn't know about the data it was trained.

### DISCUSSION THREAD

**Discussion by venturaEffect** [huggingface/autotrain-advanced#439] [Q&A]
Hi,

I'm still working on it. It looks really promising. But, after doing the Autotrain with the data and making some merge for after converting in .gguf format. I could test it. And what I've discovered after all the work is that it doesn't know anything about the data it was trained.

I've done the data as suggested in a csv file where there is the instruction and the output and all in a columna as "text". I've run the autotrain app and have followed all the steps. It created the finetuned model.

Could not test it at this point because the template that it comes in the Readme of the finetuned model with Autotrain seems not working. I've done the merge with the base model and convert it into .gguf format. When running it on LM Studio it works but it has no clue about the data it was finetuned. It is like it never learnt about it.

I've asked to the creator of the space merge-lora and he has told me there has to be a problem on finetuning.

Any idea why it doesn't show anything of knowledge of what it was finetuned?

Appreciate

**Comment by venturaEffect** [comment]
Not sure why you have moved this to discussion @abhishekkrthakur . To me it looks more a bug or maybe something wrong on running it. 

But not something for open a discussion.

**Comment by fatihayUtopia** [comment]
ı also need answer about that mr  @abhishekkrthakur. We can fine tune a model technically but can not in content
[train3.csv](https://github.com/huggingface/autotrain-advanced/files/13830006/train3.csv)
[train2.csv](https://github.com/huggingface/autotrain-advanced/files/13830008/train2.csv)
[train.csv](https://github.com/huggingface/autotrain-advanced/files/13830009/train.csv)

ı tried autotrain mistralai and mistralai instruct. with the datasets above but after autotrain them ı can not take an answer as ı expected

**Comment by abhishekkrthakur** [comment]
It seems to be working fine for me and others. If you have issues, you will need to explain properly what the issue is? what is the model trained on? what is the input and corresponding output and expected output? LLMs dont work out of the box. You need to prompt both the training data and inference data correctly in order to make it learn and reproduce learned results. For example, I have successfully finetuned mistral and other models on data with language other than english and it has always worked fine.
